content/influxdb/cloud-dedicated/get-started/query.md

```diff
@@ -338,36 +371,74 @@ _If your project's virtual environment is already running, skip to step 3._
     print(reader.to_pandas().to_markdown())
     ```
 
-    The sample code does the following:
+    {{% note %}}
```

Comment:
We could do this as an `expand` block and put `IMPORTANT` text in the expand title.

```
{{< expand-wrapper >}}
{{% expand "<span class='req'>Important</span>: If using **Windows**, specify the **Windows** certificate path" %}}

...

{{% /expand %}}
{{< /expand-wrapper >}}
```

It would look something like this:

<img width="680" alt="image" src="https://github.com/influxdata/docs-v2/assets/181484/abe03c3a-a084-4551-b04e-36d76c2ce09a">

Reply:
> We could do this as an `expand` block and put `IMPORTANT` text in the expand title.
> 
> ```
> {{< expand-wrapper >}}
> {{% expand "<span class='req'>Important</span>: If using **Windows**, specify the **Windows** certificate path" %}}
> 
> ...
> 
> {{% /expand %}}
> {{< /expand-wrapper >}}
> ```
> 

Good idea. I'll try it. I was trying to use tabs, but couldn't make it all work with the bullet nesting.

Reply:
Also, if you still want it to stand out more, but hide the content for users that don't need it, you could include an `expand` block inside of a `note` block.

<img width="573" alt="image" src="https://github.com/influxdata/docs-v2/assets/181484/913e2f13-6d4c-4a5d-9ca5-aae98008de1f">